src/Test.UnitTests.Sarif/TestData/PrereleaseCompatibilityTransformer/ExpectedOutputs/ThreadFlowLocationWithKind.sarif

```diff
@@ -0,0 +1,266 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.1.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "PREfast",
```

Comment:
> "name": "PREfast", [](start = 9, length = 19)

i added the file from the mail here, since we dont have any tests to verify this. Although, i renamed the file for easier readability.